samples/NetCoreSample/CoreApp/CoreApp.csproj

```diff
@@ -17,33 +17,61 @@
     <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
     <NuGetTargetMoniker>DNXCore,Version=v5.0</NuGetTargetMoniker>
     <BaseNuGetRuntimeIdentifier>win7</BaseNuGetRuntimeIdentifier>
```

Comment:
By default I wanted to keep these the default windows, which is why I also just kept Debug|Release targeting windows as well without renaming them. This change was more about showing what is possible but I might do further clean-up of it later.